templates/data/opensearch/index.html

```diff
@@ -0,0 +1,506 @@
+          </div>
+          <div class="col-6">
+            <p>
+              You can use OpenSearch in multiple analytics solutions such as events analytics, trace analytics, and machine learning, which uses algorithms such as anomaly detection and data clustering.
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">Observability and reporting</h3>
+          </div>
+          <div class="col-6">
+            <p>
+              OpenSearch can be used to create observability applications through the OpenSearch Dashboard. It can also be used to schedule, export and share reports.
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">Vector databases</h3>
+          </div>
+          <div class="col-6">
+            <p>
+              Use OpenSearch to store and index vectors and metadata, unlocking the ability to use low-latency queries to discover assets by degree of similarity.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <p class="p-heading--2">
+        <a href="/blog/author/mtabirao">Updates from the product manager&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </section>
+
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Learn more about Opensearch</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="/blog/vector-database-large-language-models-opensearch">Large Language Models (LLMs) Retrieval Augmented Generation (RAG) using Charmed OpenSearch</a>
+          </h3>
+          <p>
+            Retrieval-augmented generation (RAG) is a method that enables users to converse with data repositories. It can improve data retrieval, enhance knowledge sharing, and enrich the results of your LLMs to give more contextually relevant, insightful responses.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="https://ubuntu.com/engage/vector-databases-generative-ai">Vector databases for generative AI applications</a>
+          </h3>
+          <p>Improving the relevance of your LLM application by leveraging Charmed Opensearch’s vector database</p>
+        </div>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <h3 class="p-heading--5 u-no-margin--bottom">
+            <a href="/data/opensearch#get-in-touch">Discuss your needs with our team</a>
```

Comment:
Same here, can we use the class to provoke the model and have this like to a static contract us please